src/views/applicationManager.js

```diff
@@ -1,24 +1,56 @@
 const interview = {};
-interview.navigate = function(page) {
-    const appEl = document.querySelector('#app');
-    switch (page) {
-    case 'evaluations':
-        appEl.innerHTML = interview.EvaluationsPage();
-        interview.AddEventsNavigation();
-        //evaluationsEvents();
-        break;
 
-    case 'newEvaluation':
-        appEl.innerHTML = interview.newEvaluationPage();
-        interview.formEventsSetup();
-        interview.AddEventsNavigation();
-        break;
+// find page
+page = [];
+if (page == 0) {
+    page = 'login';
+} 
+else {
+//   destroy(lastPage);
+//    init(newPage);
+}
+//modules
```

Comment:
Don't get me wrong, I absolutely :heart: comments in code, since they bring extra, sometimes much needed clarity to what is going on, making ramp-up on a particular piece of code easier and less time consuming. However, in this particular case, this comment doesn't come with any added value, so might as well either skip it or improve it by making it a tad more descriptive, e.g., `// set up the collection of modules that the app can use / is aware of / has knowledge of`